sdk/master_board_sdk/example/example.cpp

```diff
@@ -89,6 +89,10 @@ int main(int argc, char **argv)
 				robot_if.PrintADC();
 				robot_if.PrintMotors();
 				robot_if.PrintMotorDrivers();
+				if (robot_if.IsTimeout())
```

Comment:
Should the `while (1)` loop of the example change to `while(!robot_if.IsTimeout())`?

Reply:
This is mostly for other people following the example code. If there is a timeout, I think the program should just stop and report an error message to the user.